src/Components/Card.tsx

```diff
@@ -111,10 +111,23 @@ export const Card = ({
   title,
   shortDescription,
   image,
-  cost,
+  costValue,
+  costQualifier,
   age,
   categories,
 }: ServicePreview): JSX.Element => {
+  const theCats = [
```

Comment:
:cat: :smile_cat: :joy_cat:

Reply:
I meant to rename, but alas, the auto-merge!